### DISCUSSION TITLE
Custom attribute on model in column

### DISCUSSION THREAD

**Discussion by wr0nggg** [rappasoft/laravel-livewire-tables#1237] [Q&A]
Is it possible to add computable value to column? i am trying to avoid calling db to get the model and then getting custom attribute inside format callback, thanks

**Comment by lrljoe** [comment]
Can you give me an example of how you're using a computed value please and I'll see if I can replicate it on my end.

**Comment by wr0nggg** [comment]
just stupid example: i have User table with birth date, computed attribute is return whether the user is kid, teen, adult
class User {
getCustomAttribute{
if (now - date_of_birth < 5) return kid
}
then in datatable of users i want to call user->custom

**Comment by emiliosh** [comment]
Not sure if you are still interested but I found a solution here:

#https://github.com/rappasoft/laravel-livewire-tables/discussions/831

**Comment by lrljoe** [comment]
You can also use the setTdAttributes to add attributes to a particular column.

It is in the docs, but shout if it isn't clear.
